app/models/phi_x.rb

```diff
@@ -55,7 +55,7 @@ def self.tag_options
   # Returns the names of valid tag options for creation of PhiX libraries.
   # @return [Array] Valid tag options
   def self.tag_option_names
-    tag_options.keys
+    tag_options.keys.map(&:to_s)
   end
```

Comment:
A lot of the changes here are due to the way rails 6 handles config loading. Fortunately this is all backwards compatible.